virttest/qemu_devices/qcontainer.py

```diff
@@ -2057,8 +2061,13 @@ def define_hbas(qtype, atype, bus, unit, port, qbus, pci_bus, iothread,
                                  "on %s by -blockdev." % (opt, name))
             if media == 'cdrom':
                 readonly = 'on'
-            protocol_node.set_param('read-only', readonly, bool)
             format_node.set_param('read-only', readonly, bool)
+
+            # Set auto-read-only/discard to keep consistent with libvirt
```

Comment:
Then, seems we don't need this comment no more.

Reply:
Removed.